tests/pytest_tests/release_tests/test_launch/test_kubernetes.py

```diff
@@ -0,0 +1,27 @@
+import subprocess
+
+from wandb.sdk.launch.launch_add import launch_add
+
+ENTITY = "launch-release-testing"
+PROJECT = "release-testing"
+QUEUE = "kubernetes-queue"
+JOB_NAME = "job-source-release-testing-train-np.py:v0"
+
+
+def test_kubernetes_successful_run():
+    try:
+        # Start launch agent
+        agent_cmd = ["wandb", "launch-agent", "-q", QUEUE, "-e", ENTITY]
```

Comment:
This is probably worth testing, since it tests an agent on a machine submitting to k8s. But we also will want to test with the agent in k8s.

Reply:
That'll come in a followup PR, it's ending up a bit more involved than this simpler test so it'll be nicer to have it separate.